tests/ragger/test_eip2930.py

```diff
@@ -0,0 +1,62 @@
+from web3 import Web3
+
+from client.client import EthAppClient
+import client.response_parser as ResponseParser
+from client.utils import recover_transaction
+
+from ragger.backend import BackendInterface
+from ragger.firmware import Firmware
+from ragger.navigator import Navigator, NavInsID
+
+from constants import ROOT_SNAPSHOT_PATH
+
+
+BIP32_PATH = "m/44'/60'/0'/0/0"
+NONCE = 21
+GAS_PRICE = 5
+GAS_LIMIT = 21000
+ADDR = bytes.fromhex("b2bb2b958afa2e96dab3f3ce7162b87daea39017")
+AMOUNT = 0.01
+CHAIN_ID = 3
+
+
+def test_sign_eip_2930(firmware: Firmware,
```

Comment:
Maybe move it in test_sign.py since all the transaction signature tests are there already.

Reply:
Done